### ISSUE TITLE
Poor legibility for read only events

### ISSUE THREAD

**Issue by camaz** [nextcloud/calendar#555]
I subscribe to an .ics internet calendar which is read only. When I click on 'More' and view the details on the right of the calendar, the text is very difficult to read because of the low contrast. I'm sure there is another way to display this so the user knows it is read only, while maintaining the legibility of the content.

Thanks for the great app!

**Comment by alexanderdd** [comment]
Hi!
Same problem but different use case: I publicly share a calendar via http link, and include that in a website with an iframe. All the information in the pop-up is grey text on grey background -- hard to read.

Thanks to everyone on the team!

**Comment by georgehrke** [comment]
@tcitworld Any progress on this? :)

**Comment by Berni-MD** [comment]
Is it maybe possible somewhere to settig by css?

**Comment by georgehrke** [comment]
@nextcloud/designers Should we fix this in calendar or in the Nextcloud server?

![1ab4fc3c-7ea9-40ea-b692-3e34028e623c](https://user-images.githubusercontent.com/1250540/37092529-8c849fc8-220c-11e8-8c77-d8e09da83839.jpg)

**Comment by ichbestimmtnicht** [comment]
I would be neat to allow copying of at least the description of the subscribed calendar to inform friends or colleagues without a proper software about some events. Like the Meteor shower info from https://cantonbecker.com/astronomy-calendar-files/astrocal.ics today. All entries are greyed out, can't be marked neither copied....

I am using Nextcloud hosted on cloud.disroot.org (it's not a point of interest for this i think, but before you ask me for serverlogs....)

**Comment by jancborchardt** [comment]
Read-only content in this case should _not_ be displayed via a disabled input field. Instead simply render it as text. Of course we need to make sure the elements have the same layout as the inputs.

cc @skjnldsv since you brought it up in IRC. :)

**Comment by alexanderdd** [comment]
Hey everyone, this issue was reported a year ago, and by now already five duplicate bugs have been filed. I think this shows that the users want a better UX for read-only events and read-only shared calendars. 

The development of this feature was set for 1.6.2., then for 1.7.0 and now for 2.0.0. beta1. I am a little worried that no development will happen and it will be rescheduled for the following version again.

It also seems relatively clear what needs to happen: render info as regular text, not as disabled input field. (see @jancborchardt above)

Is there anything I/we can do to help bring this along? (I can't do the coding unfortunately.. but I could ask for coding support on the nextcloud forums or reddit?)

**Comment by jancborchardt** [comment]
Yes @alexanderdd, you and everyone requesting this (and any other fix or feature) can help apart from developing – as it says in the original issue post:

**[💸 Donate to the bounty on this issue](https://www.bountysource.com/issues/47917181-poor-legibility-for-read-only-events)** (it’s currently at $ 0)

**Comment by Dubidubiduu** [comment]
I support the necessity to use a proper contrast! I also want to ask, if it is possible to get the details of an event on the first click on the event without using the "more" button.

Cause it is necessary for everyone of my ppl to get the details of an event and an additional click lower the rates of ppl get that information.

**Comment by mcnesium** [comment]
Also, if there are links in the description, they are not clickable: 

<img src="https://user-images.githubusercontent.com/476060/57304878-61f5e180-70e0-11e9-855f-2925527764f3.png" alt="Screenshot" width="459" height="479">

I would suggest to remove the text areas on read-only events and display plain text instead. Or you could even support markdown, how great would that be :astonished:

**Comment by pm-pm** [comment]
I opened #1691 because i could not believe that problem exists over years.
I have also found some other restrictions at the calender app: Using on smaller screens have problems.

Does anybody use the calender/nextcloud in real production with real users who are allowed to complain? ;-)
If "yes" the user would complain loudly. 

I know it is not easy to program all the code, but please make the software useable for the day by day work. Thanks. :-)

**Comment by georgehrke** [comment]
@pm-pm This bug has been fixed, that's why it's closed :) 

> Using on smaller screens have problems.

Please open a dedicated issue with a full problem description. If you comment on closed issues with "oh btw, feature abc also doesn't work like expected" your comment just gets lost and never fixed.
